xrviz/dashboard.py

```diff
@@ -20,23 +20,28 @@
 class Dashboard(SigSlot):
     """
     This section provides access to the complete generated dashboard,
-    consisting of all subsections.
+    consisting of all the panes.
 
     Parameters
     ----------
-    data: `xarray` instance: `DataSet` or `DataArray`
-           datset is used to initialize.
-    initial_params: To pre-set values of widgets.
+    data: `xr.core.dataarray.DataWithCoords`
```

Comment:
As above. I believe the other cases take only `xarray.DataSet`. Note that you should not abbreviate with `xr`.